agent/bench-scripts/pbench-uperf.md

```diff
@@ -57,11 +57,11 @@ Finally, if you want to simulate a mixed read-write workload, use these pairs:
 
     (c1,s1), (c2, s2), (c3, s1), (c4, s2), (s1, c1), (s2, c2), (s1, c3), (s2, c4)
 
-More complicated flows can model behavior of non-native protocols, 
-where a cluster node acts as a proxy server - 
-it is a server (for non-native protocol) and a client (for native protocol). 
-For example, such protocols often induce full-duplex traffic 
-which can stress the network differently than unidirectional in/out traffic. 
+More complicated flows can model behavior of non-native protocols,
+where a cluster node acts as a proxy server -
+it is a server (for non-native protocol) and a client (for native protocol).
+For example, such protocols often induce full-duplex traffic
+which can stress the network differently than unidirectional in/out traffic.
 For example, try adding this set of flows to preceding flow:
 
     (s1, s2),.(s2, s3),.(s3, s4),.(s4, s1)
```

Comment:
Not relevant to your changes, but this line happens to be within GitHub's view, and is the notation here a bit odd... or does the `,.` mean something specific in context that I'm not getting?

Reply:
I think that's a typo and I meant to fix it but I forgot to do it. I might as well fix it.